### ISSUE TITLE
Add email-renderer app

### ISSUE THREAD

**Issue by ocodista** [ocodista/trampar-de-casa#79]
All email templates we have (confirmation and openings) were created using react.email website.

It would be nice to have one app to be able to render the templates inside this project.

Ideally, it would be a "copy" version of the react-email, but it should only display our templates.

**Comment by Fritola** [comment]
I'm working on it!

**Comment by ocodista** [comment]
@Fritola Thank you!

Assigned, please let us know of the advances or blockers if possible

**Comment by Fritola** [comment]
I had success bringing the preview from the react-email and rendering an example from it. Now I'm blocked by trying to render our emails

**Comment by albertofp** [comment]
@Fritola Any progress on this issue? I'd like to take a look at it myself but I'm not sure how far you made it

**Comment by ocodista** [comment]
@albertofp Please assume this one

**Comment by albertofp** [comment]
My bad, had a lot going on and didn't have the time to work on this. Feel
free to reassign to someone else

**Comment by emvictor** [comment]
Any news on this issue? I'd like to give it a try this week.

**Comment by ocodista** [comment]
@emvictor go for it, we still need it!
